### PR TITLE
Forms 

### PR DESCRIPTION
**Pull request by Ceci007**
## Bare Metal Forms and Helpers
In this project we:
- Add a User model.
- Add a User controller.
- Add views for the user and the form partial to create and edit users.
- Integrate bootstrap and add some helpers.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Rails.application.routes.draw do
+  get 'signup', to: 'users#new'
```
Comment (christianotieno) [inline config/routes.rb:2]: ## Improve code on the routes config
- [ ] let's make the signup page as the default (`root`) route

File: config/routes.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+  resources :users, except: [:new]
```
Comment (christianotieno) [inline config/routes.rb:3]: ## Restrict restful routes
- [ ] Restrict these restful resources as advised below. 👇 
We want to only have relevant routes for our app.([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_route1.png) [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_route2.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))

```suggestion
  resources :users, only: [:new, :create, :edit, :update]
```

**Review by christianotieno — CHANGES_REQUESTED** [review]
# Changes Required ♻️ 
Hello team, 👋 good job working on the project.
We do have some few issues to take care of before we proceed below 👇 

### Happy coding 💻 
#### Christian Otieno (TSE)

**Review by MuriloRoque — APPROVED** [review]
# Status: Approved :green_circle: 

### Hello @Ceci007 and @happiguru :wave: 

### Thank you for making all of the required changes, your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**